db_release_scripts/mosip_master/sql/1.2.0_master-scripts_release.sql

```diff
@@ -255,6 +260,16 @@ ALTER TABLE master.user_detail_h DROP COLUMN uin;
 ALTER TABLE master.user_detail_h DROP COLUMN email;
 ALTER TABLE master.user_detail_h DROP COLUMN mobile;
 
+SELECT * INTO master.template_copy FROM master.template;
```

Comment:
If for backup, please use the bkp convention to be consistent everywhere

Reply:
take care of this and below in the revoke script also

Reply:
Done

Reply:
In one way it make sense, but now admin will not be consistant with all the other modules. Also in some cases, the full db backup will take a very long time and there might be large table which does not have any change.

Reply:
Lets remove the whole db backup point from documentation

Reply:
updated the document about revoke script and also about the backups taken.